### ISSUE TITLE
Problem with @rollup/plugin-commonjs@16.0.0

### ISSUE THREAD

**Issue by frederikhors** [PepsRyuu/nollup#159]
Issue on @rixo's `svelte-template-hot`: https://github.com/rixo/svelte-template-hot/issues/16.

From `"@rollup/plugin-commonjs": "16.0.0"` this is the error:

```
[Nollup] Listening on http://localhost:5000
TypeError: Cannot read property 'length' of undefined
    at new MagicString (C:\svelte\node_modules\magic-string\dist\magic-string.cjs.js:354:34)
    at module.exports (C:\svelte\node_modules\nollup\lib\impl\ImportExportResolver.js:318:21)
    at compileModule (C:\svelte\node_modules\nollup\lib\impl\NollupCompiler.js:59:30)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async compileInputTarget (C:\svelte\node_modules\nollup\lib\impl\NollupCompiler.js:138:5)
    at async Object.compile (C:\svelte\node_modules\nollup\lib\impl\NollupCompiler.js:211:31)
    at async generateImpl (C:\svelte\node_modules\nollup\lib\index.js:13:21)
```

Reproduction here: https://github.com/frederikhors/svelte-template-hot/tree/update-commonjs-to-16

Maybe a nollup problem?

Thanks for your work.

**Comment by PepsRyuu** [comment]
Seems very likely! Will have a chance to look over it this weekend. Thanks for the report!

**Comment by PepsRyuu** [comment]
https://github.com/PepsRyuu/nollup/pull/160/files This branch should solve the problem, would appreciate if you could confirm from your end as well before I release. :)

**Comment by frederikhors** [comment]
I downloaded the files of that branch, copied `lib` in my project's `node_modules/nollup/lib` and it works!

**Comment by rixo** [comment]
You rox guys!

**Comment by PepsRyuu** [comment]
Released in ```0.14.0```. :)
